scripts/assemble-manifest.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-extraneous-dependencies */
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs-extra');
+const { getPackages } = require('@manypkg/get-packages');
+const path = require('path');
+
+async function main() {
+  const [version, outputPath] = process.argv.slice(2);
+  if (!version || !outputPath) {
+    throw new Error('Argument must be script.js <version> <outputPath>');
```

Comment:
Is `process.argv[0]` or 1 a sensible string, instead of script.js?

Reply:
Good point! `argv[1]` is going to be the full path to the script but it makes more sense then script.js